### PR TITLE
Adding support for https in check-influxdb

### PR DESCRIPTION
**Pull request by leelasharma**
Summary:
Adding support for https in check-influxdb.
Passing ssl arguments to the influxdb object  in check-influxdb-query.
Renaming the config option in check-influxdb from ssl to verify_ssl, so its consistent.
Changing README to update the config option ssl_ca_cert.

### CODE REVIEW

File: bin/check-influxdb.rb [hunk 1]
```diff
@@ -28,9 +28,7 @@
 #
 
 require 'sensu-plugin/check/cli'
-require 'net/http'
-require 'uri'
-require 'json'
```
Comment (tas50) [inline bin/check-influxdb.rb:33]: You'll want to remove these from the header block as well

Comment (leelasharma) [inline bin/check-influxdb.rb:33]: Thanks @tas50 for the note. Done!

**Comment by mattyjones** [comment]
@leelasharma  Can please update the CHANGElog as well.  Thanks!

**Comment by leelasharma** [comment]
@mattyjones Done. Also, does the version.rb  file need to be manually updated or is it taken care during release?

**Comment by mnuessler** [comment]
Is there anything left to do before this can be merged? Would be nice to have support for HTTPS..

**Comment by mattyjones** [comment]
@mnuessler @leelasharma [0.0.5](https://rubygems.org/gems/sensu-plugins-influxdb/versions/0.0.5)

**Comment by leelasharma** [comment]
Thanks!

**Pull request merged**